docs/portal/developer-portal/uan_admin_guide.ditamap

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
 <map id="uan_admin">
-    <title>HPE Cray Supercomputing User Access Node (UAN) Software Administration Guide (S-8033)</title>
+    <title>HPE Cray Supercomputing User Access Node (UAN) Software Administration Guide (S-8033) @product_version@</title>
```

Comment:
this should probably be `@product_version_short@`

Reply:
We want the "X.Y.Z" version here.